examples/src/bunnymark/mod.rs

```diff
@@ -366,11 +373,11 @@ impl crate::framework::Example for Example {
 
     fn resize(
         &mut self,
-        _sc_desc: &wgpu::SurfaceConfiguration,
+        sc_desc: &wgpu::SurfaceConfiguration,
         _device: &wgpu::Device,
         _queue: &wgpu::Queue,
     ) {
-        //empty
```

Comment:
It would be good to recompute the orthographic matrix too, which is computed on line 277:
```
        let globals = Globals {
            mvp: glam::Mat4::orthographic_rh(
                0.0,
                config.width as f32,
                0.0,
                config.height as f32,
                -1.0,
                1.0,
```
and also update the GPUBuffer with `queue.writeBuffer`:
```
        let global_buffer = device.create_buffer_init(&wgpu::util::BufferInitDescriptor {
            label: Some("global"),
            contents: bytemuck::bytes_of(&globals),
            usage: wgpu::BufferUsages::COPY_DST | wgpu::BufferUsages::UNIFORM,
        });
```

otherwise the matrix will be fixed to the original window size which seems incorrect.

Reply:
thats what I'm planning to do but there are few other things that depend on this so i'd have to extract it all out to reuse it